static/js/src/advantage/subscribe/reducers/form-reducer.js

```diff
@@ -5,6 +5,15 @@ const isSmallVP =
   Math.max(document.documentElement.clientWidth || 0, window.innerWidth || 0) <
   875;
 
+const productsArray = Object.entries(window.productList);
+function hasMonthly() {
```

Comment:
We should check if the selected product has a monthly option not just if any do.

Reply:
You're right it makes more sense this way. I updated it